EllipticCurve.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+
+  s.name         = 'EllipticCurve'
+  s.version      = '0.3.3'
+  s.summary      = 'ECDSA in pure Swift'
+
+  s.description  = <<-DESC
+                  Elliptic Curve Digital Signing Algorithms in pure Swift
+                   DESC
+
+  s.homepage     = 'https://github.com/hyugit/EllipticCurve.git'
+  s.license      = 'MIT'
+  s.authors      = { 'Huang Yu' => '13056774+mryu87@users.noreply.github.com', 'Alexander Cyon' => 'alex.cyon@gmail.com' }
+  s.source = { :git => 'https://github.com/Sajjon/EllipticCurve.git', :branch => 'cocoapods' }
```

Comment:
also here, if you can change it to 'https://github.com/hyugit/EllipticCurve.git', it should work.